src/ripple/app/misc/ValidatorList.h

```diff
@@ -125,6 +125,14 @@ class ValidatorList
 
     PublicKey localPubKey_;
 
+    // This is the minimum quantity of validators from which a resizeable
+    // trusted set can be created. In other words, if the size of the list of
+    // all validators is less, then use all of them in the trusted set.
+    int const MINIMUM_RESIZEABLE_UNL {24};
+    // Maximum number of total validators to require no more than Byzantine
+    // fault tolerance.
+    int const MAXIMUM_BYZANTINE {32};
```

Comment:
The comment and the name are a bit confusing. The maximum number of something to require no more than something? I just have a hard time parsing it.

How about renaming this to `BYZANTINE_THRESHOLD` and changing the comment to something like:

>The maximum size of a trusted set for which Byzantine fault tolerance isn't needed.

Or
>The minimum size of a trusted set that requires Byzantine fault tolerance.